.travis.yml

```diff
@@ -3,13 +3,9 @@ language: ruby
 sudo: false
 
 rvm:
-  - 1.9.3
-  - 2.0.0
   - 2.1
-  - 2.2
   - jruby-19mode
-  - rbx-2
-  - ruby-head
+  - jruby-9.0.4.0
```

Comment:
The only issue we saw were some randomly failling specs that I assume were due to some caching issues in the specs (this might be fixed upstream though). After retrying them, the specs would then pass.

As a side note, we haven't use `jruby-9000` in production yet. We might do so in the first quarter of 2016 but in development AMS works pretty fine.